src/main/java/com/github/alina/session_17_generics/challenge/xml_to_json/BookStore.java

```diff
@@ -0,0 +1,26 @@
+package com.github.alina.session_17_generics.challenge.xml_to_json;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+
+import java.util.List;
+
+public class BookStore {
+    @JacksonXmlElementWrapper(useWrapping = false)
+    private List<Book> book;
```

Comment:
Please pay attention to the name of the properties. When we use List<>, we can consider names ending with `...List` or the plural of the property.

Reply:
Ok, thanks.